pages/admin/dashboard.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components'
```

Comment:
We might want to save `/admin` for lvl/IndieDAO admin, and use something like `/dao` for community managers to login to.